bin/gen_csr_access_test.py

```diff
@@ -17,34 +17,55 @@
     default_core = 'None'
 
 topdir = os.path.abspath(os.path.join(os.path.dirname(os.path.realpath(__file__)), '..'))
+
+
+# Parser arguments
+
 parser = argparse.ArgumentParser()
+
 parser.add_argument("--dry_run",                help="Prints generated yaml to stdout",   action="store_true")
 parser.add_argument("--core",                   help="Set the core to generate test for", choices=supported_cores)
+parser.add_argument("--output",                 help="Output path",                       type=str)
+parser.add_argument("--iterations",             help="Number of generated tests",         type=str, default="1")
+parser.add_argument("--m4",                     help="Use 'm4' for all preprocessing",    action="store_true", default=False)
+
 parser.add_argument("--pmp_num_regions",        help="Set number of Pmp regions",         type=str, default="0")
 parser.add_argument("--mhpmcounter_num",        help="Set number of mhpmcounters",        type=str, default="3")
 parser.add_argument("--num_triggers",           help="Set number of trigger registers",   type=str, default="1")
+parser.add_argument("--marchid",                help="Set number for marchid",            type=str, default="0")
+
 parser.add_argument("--clic_enable",            help="Enable clic support",               action="store_true", default=False)
-parser.add_argument("--zc_enable",              help="Enable Zc support",                 action="store_true", default=False)
 parser.add_argument("--clint_enable",           help="Enable Basic Interrupts",           action="store_true", default=False)
 parser.add_argument("--debug_enable",           help="Enable Debug Registers",            action="store_true", default=False)
+parser.add_argument("--umode_enable",           help="Enable Certain U-mode fields",      action="store_true", default=False)
```

Comment:
Mostly just new conditional names to support more features from the user manuals.